packages/core/src/workflow/dealer-queue.js

```diff
@@ -16,7 +16,7 @@ import { decode as aggregateDecode } from '../data/aggregate.js'
  * @param {import('@web3-storage/filecoin-client/types').InvocationConfig} props.invocationConfig
  * @param {import('@ucanto/principal/ed25519').ConnectionView<any>} props.dealerServiceConnection
  */
-export async function dealerAdd ({
+export async function dealerQueue ({
```

Comment:
rename to what this actually does, which is queuing the deal for the dealer to handle